Geometry/CommonTopologies/interface/SimplePixelTopology.h

```diff
@@ -216,7 +218,7 @@ namespace phase2PixelTopology {
   using pixelTopology::phi0p07;
   using pixelTopology::phi0p09;
 
-  constexpr uint32_t numberOfLayers = 28;
+  constexpr uint32_t numberOfLayers = 28 + 3;  // CA Extension with 3 barrel layers from OT
```

Comment:
It looks like this change to `numberOfLayers` was not propagated to `layerStart` (line 246), `dcaCuts` (line 305), and `thetaCuts` (line 310).